### PR TITLE
Added additional common environment variables that may need to be set to CLI help menu.

### PR DESCRIPTION
**Pull request by JamesDrummond**
### What does this PR do?
Added additional common environment variables that may need to be set to CLI help menu.

### What issues does this PR fix or reference?
NA

#### Release Notes
Added additional common environment variables that may need to be set to CLI help menu.

Signed-off-by: James Drummond james@devcomb.com

### CODE REVIEW

**Review by riuvshin — APPROVED** [review]

**Comment by JamesDrummond** [comment]
Waiting for one more approval.

**Comment by JamesDrummond** [comment]
@benoitf Can you approve this PR?

**Comment by JamesDrummond** [comment]
@sleshchenko Can you approve this PR?

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**
